policyengine_us/variables/gov/hhs/tanf/cash/eligibility/is_person_demographic_tanf_eligible.py

```diff
@@ -9,7 +9,10 @@ class is_person_demographic_tanf_eligible(Variable):
 
     def formula(person, period, parameters):
         child_0_17 = person("is_child", period)
-        is_18 = person("age", period) == 18
+        is_18 = (
+            person("age", period)
+            == parameters(period).gov.hhs.tanf.cash.eligibility.age
+        )
         full_time_student = person("is_full_time_student", period)
```

Comment:
do we have a variable to indicate secondary school? i think `is_full_time_student` includes college which doesn't apply to tanf

Reply:
no we don't, i will add one